### ISSUE TITLE
Error when giving "Zn" in elementlist

### ISSUE THREAD

**Issue by MadFisa** [chianti-atomic/chiantipy#378]
I am getting the following error when running the code given below. The error is as follows"

Process Process-21:
Traceback (most recent call last):
  File "/usr/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/dist-packages/ChiantiPy/tools/mputil.py", line 24, in doFfQ
    ff = ChiantiPy.core.continuum(ionS, temperature, abundance=abund, em=em)
  File "/usr/local/lib/python3.7/dist-packages/ChiantiPy/core/Continuum.py", line 90, in __init__
    self.Ip = chdata.Ip[self.Z-1, self.Stage-1]
IndexError: index 30 is out of bounds for axis 1 with size 30

The code is 

```
import ChiantiPy.core as ch
import ChiantiPy.tools.io as chio
import matplotlib.pyplot as plt
import numpy as np


elements_list = [ "Zn"]
logT = [5.6, 6, 6.3, 6.5, 7, 7.1]
em = [1e26] * len(logT)
E = np.linspace(0.1, 15, 1000)
conversion_factor = 12.398
wvl = conversion_factor / E
density = 1e9
elements = elements_list[:30]
#%% Runs Chiantipy
element_list = ["Zn"]
s = ch.mspectrum(np.power(10.,logT),dens,wvl,elementList=element_list,em=em,verbose=1)
)
```

I am unable to figure out what is causing the issue. My guess is that this particular way is trying to do calculation for zn_31 ion, which probably does not exist in the database. Please advice on how to fix this. Could be related to #358 .

Asif

**Comment by kdere** [comment]
The problem is that it is trying to find the ionization potential of the of the bare Zn ion.  This does not exist.  I will look into  how to fix it.
Ken

**Comment by kdere** [comment]
this has been fixed in the  0.14.0 release
